### PR TITLE
Get more information about oids that create errors

### PR DESCRIPTION
**Pull request by pbauer**
1. List all oids for a specific error
2. New feature: inspect a single oid by passing it to zodbverify. The idea is to run zodbverify on the whole database and from the output copy one oid and run it again to further inspect that object.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -45,14 +45,37 @@ The verification runs in the context of the initialized Zope application.
 Usage::
 
     ./bin/instance zodbverify [-h] [-D]
-
```
Comment (dwt) [inline README.rst:48]: It seems this is missing documentation of the new flag?

**Review by dwt — COMMENTED** [review]
For the rest of the pull request I can't really comment on, as I don't have intimate enough knowledge. :/

**Comment by jensens** [comment]
LGTM, just the doc-thing mentioned above.

**Review by dwt — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (mauritsvanrees) [reply README.rst:48]: I have updated the documentation by pasting the new help output.

**Review by mauritsvanrees — APPROVED** [review]

**Pull request merged**